backend/api/src/main/kotlin/software/shonk/application/service/ShorkService.kt

```diff
@@ -15,7 +15,7 @@ class ShorkService(private val shork: IShork) : ShorkUseCase {
     private var lobbyCounter = 0L
 
     init {
-        createLobby()
+        createLobby("default")
```

Comment:
The goal of these endpoints in v1 is to get rid of the default lobby. Therefore this can be removed. Tests need to be adjusted accordingly.

The new intended behaviour for this milestone is, that there is no lobby by default and we have to create them one by one by hand.